### ISSUE TITLE
Please add PDF Mod

### ISSUE THREAD

**Issue by Valeryan24** [abandonware/abandonware.github.io#4]
Hi,

Pdf Mod was a very useful Linux program allowing to work with pdf documents : merge, split, add or remove some pages, bookmarks management (which needs improvement, like moving them or insert one between others) etc...

Working with Mono. Development stopped since 2011, and there were many important bugs not fixed (Bugzilla Gnome 609278, 628440, 651377, 651519, 651973, 655654, 597760 ). Code should also need adaptation to new libraries. It would be great if someone could fork it and maintain again.

Prove of abandonware :
https://bugzilla.gnome.org/show_bug.cgi?id=651519#c11

Archived by Gnome here :
https://gitlab.gnome.org/Archive/pdfmod/commits/master

Thanks :)

**Comment by rzr** [comment]
Does it relate to @dineshkummarc ?
https://github.com/dineshkummarc/pdfmod


It can be eventually transferred to :
https://gitlab.com/abandonware or https://github.com/abandonware/

May you request this to current maintainers if any ?

BTW I use pdftk for this kind of operations...

More feedback welcome

**Comment by Valeryan24** [comment]
Yes, same program. But except translations, last commit is from March 2011, not developed anymore.

As it is on Github, can't it be just forked ?

I also use PDF Arranger, fork of PDF Shuffler, but it has no bookmarks management and I don't think pdftk / Pdf Chain have it, too (add and edit bookmarks)...

**Comment by rzr** [comment]
before forking it would be be nice to identify if there any volunteers to help on it ?

Anyway it is preferred to have it transfered by upstream as explained in the following procedure:

https://abandonware.github.io/

**Comment by rzr** [comment]
Any feedback from upstream ?
